src/components/config/create-config.ts

```diff
@@ -0,0 +1,40 @@
+/**********************************************************************************
+ * MIT License                                                                    *
+ *                                                                                *
+ * Copyright (c) 2021 Hyperjump Technology                                        *
+ *                                                                                *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy   *
+ * of this software and associated documentation files (the "Software"), to deal  *
+ * in the Software without restriction, including without limitation the rights   *
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell      *
+ * copies of the Software, and to permit persons to whom the Software is          *
+ * furnished to do so, subject to the following conditions:                       *
+ *                                                                                *
+ * The above copyright notice and this permission notice shall be included in all *
+ * copies or substantial portions of the Software.                                *
+ *                                                                                *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR     *
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,       *
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE    *
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER         *
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,  *
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE  *
+ * SOFTWARE.                                                                      *
+ **********************************************************************************/
+
+import fs from 'fs'
+import axios from 'axios'
+import { log } from '../../utils/pino'
+
+export const createConfigFile = async (filename: string): Promise<void> => {
+  log.info(`No Monika configuration available, initializing...`)
+
+  const url =
+    'https://raw.githubusercontent.com/hyperjumptech/monika/main/monika.example.yml'
+  await axios.get(url).then((resp) => {
+    fs.writeFileSync(filename, resp.data, 'utf-8')
+  })
+  log.info(
+    `monika.yml file has been created in this directory. Change the URL in the probe section to your URL.`
```

Comment:
It seems like the filename won't always be monika.yml. It should be 

```
`${filename} file has been created in current directory.`
```

Also 

```
You can change the URL to probe and other configurations in that ${filename} file.
```

Reply:
done